go/vt/vttablet/tabletserver/twopc_test.go

```diff
@@ -396,3 +397,73 @@ func jsonStr(v any) string {
 	out, _ := json.Marshal(v)
 	return string(out)
 }
+
+// TestUnresolvedTransactions tests the UnresolvedTransactions method.
```

Comment:
the comment should say something about what we are testing. this comment does not say more than the test method name already says